frontend/src/metabase/nav/components/SearchResults.jsx

```diff
@@ -81,10 +82,14 @@ export default _.compose(
     wrapped: true,
     reload: true,
     debounced: true,
-    query: (_state, props) => ({
-      q: props.searchText,
-      limit: DEFAULT_SEARCH_LIMIT,
-      models: props.models,
-    }),
+    query: (_state, props) => {
+      const searchFilters = props.searchFilters || {};
+      return {
+        q: props.searchText,
+        limit: DEFAULT_SEARCH_LIMIT,
+        ...searchFilters,
```

Comment:
Is the reason you spread `searchFilters` here is you don't want only `models` to get overridden?

Reply:
Yep, this is a vestigial element of the previous search, where we used `models` for the search query. We can change that in the future but I didn't want to anything more drastic for this iteration